index.js

```diff
@@ -1 +1,38 @@
 // Code your solution in this file!
+function distanceFromHqInBlocks(num1){
```

Comment:
What do you think a better name for the argument `num1` could be?

Reply:
block or blockNum?

Reply:
Either would be good! I like `blockNumber` or even `startingBlockNumber` since it's most descriptive.